### DISCUSSION TITLE
Zooming in?

### DISCUSSION THREAD

**Discussion by bjoseph0916** [pawelsalawa/sqlitestudio#5156] [General]
Hi, how do I zoom in for SQLiteStudio? The size of the screen of the UI is pretty small for my monitor and I wanted to know if there's a way I can zoom in or render the size for the SQL editor. 

![image](https://github.com/user-attachments/assets/dccd16f1-e394-4930-a9c9-98814be88e7a)

I can even do with an option for changing the font size if there is any option for it.

**Comment by pawelsalawa** [comment]
You can change fonts in configuration:
![image](https://github.com/user-attachments/assets/c8aff23b-825d-4887-8fd9-bee415824c02)
